import/repos.json

```diff
@@ -8,6 +8,10 @@
     "currentBranch": "master",
     "repo": "https://github.com/EventStore/EventStore.git",
     "branches": [
+      {
+        "version": "v21.6",
+        "name": "docs_21.6_version_update"
```

Comment:
**TODO:** Update that to release branch once https://github.com/EventStore/EventStore/pull/2989 is merged.